testing/integration/upgrade_broken_package_test.go

```diff
@@ -32,6 +32,8 @@ func TestUpgradeBrokenPackageVersion(t *testing.T) {
 		Sudo:  true,  // requires Agent installation
 	})
 
+	t.Skip("Need to be skip until we have 8.14 first build")
```

Comment:
Why do we need to skip this now? We didn't need to do this when the 8.12 feature freeze happened IIRC.

Reply:
We need to because of the 8.13.0 agent version pinning.
What is happening is:

- We install with version 8.13.0-SNAPSHOT (as set in package version file) with a BEATS_VERSION of 8.14.0
- the install directory is `elastic-agent-8.13.0-SNAPSHOT-<hash>`
- we delete the package version file (this is what the test does on purpose)
- the fallback for agent without a package version file is to use BEATS_VERSION (which is different now)
- the agent is not capable of calculating paths correctly anymore because the install location does not match neither `elastic-agent-<hash>` nor `elastic-agent-<version>-<hash>`
- As a result we have CLI trying to connect to the wrong path for agent socket (isInsideData()) tries to match the exact install path
- watcher rollback is also broken because we think we sholud be in `elastic-agent-8.14.0-SNAPSHOT`

Reply:
Discussed with @cmacknz, we will keep it that way for now and we will try to find a more persistent solution for the future.